README.md

```diff
@@ -12,12 +12,20 @@ Chrome Platform Status
 ### Installation
 1. Before you begin, make sure that you have a java JRE (version 8 or greater) installed. JRE is required to use the DataStore Emulator.
 1. Install global CLIs
-    1. [Google App Engine SDK for Python](https://cloud.google.com/appengine/docs/standard/python/setting-up-environment). Make sure to select Python 2.7.
+    1. [Google App Engine SDK for Python](https://cloud.google.com/appengine/docs/standard/python3/setting-up-environment). Make sure to select Python 3.
     1. pip, node, npm.
     1. Gulp `npm install --global gulp-cli`
 1. Install npm dependencies `npm ci`
```

Comment:
this line should go after the `Create a virtual environment`?

Reply:
No, I think that the GAE SDK is installed on the user's machine independently of a virtual environment.  As I understand it, the venv is really just about python libraries from requirements.txt.